NAS-117098 / 22.12 / Nas 117098

AlexKarpov98 opened the pull request:
The cards for a pool summary are aligned again and the card's size is changed so that cards are aligned with an horizontal layout.

bugclerk commented:
Jira URL: https://ixsystems.atlassian.net/browse/NAS-117098

denysbutenko reviewed (changes requested):
Mockup preview:
![Mockup](https://user-images.githubusercontent.com/29865743/172702141-16271f37-1385-4a1e-9c9c-e6cba5b6bb92.png)

PR preview:

https://user-images.githubusercontent.com/351613/178310116-25d9a486-ce10-4d8d-aafd-401a0492455c.mp4

Noticed things:
- Gap between unused resources row and pool row do not match to other places.
- Gap between pool rows looks smaller than on mockups
- Pool name and pool actions is not aligned
- Unit tests are failed

AlexKarpov98 commented:
- Gap between unused resources row and pool row do not match to other places.
Fixed
![Screenshot_37](https://user-images.githubusercontent.com/22980553/178473249-bbcb283b-3815-4376-824a-8c14cf203e89.png)


- Gap between pool rows looks smaller than on mockups
Fixed to 30px from 20px
![Screenshot_36](https://user-images.githubusercontent.com/22980553/178473389-d4f81423-dcc7-490a-80e3-e90617564dca.png)


- Pool name and pool actions is not aligned
Fixed
![Screenshot_39](https://user-images.githubusercontent.com/22980553/178473531-839b9474-e525-4d1e-949f-d805c178360a.png)


- Unit tests are failed
Fixed - Added `mockCall('disk.query', [])` in dashboard-pool.component.spec.ts
![Screenshot_40](https://user-images.githubusercontent.com/22980553/178473585-ace1bb6b-cf84-41be-bcb8-32b361f9c0b2.png)

denysbutenko reviewed (approved):
Looks good
